truffle.js

```diff
@@ -1,13 +1,12 @@
 require("dotenv").config();
-const host = process.env.RPC_HOST || "127.0.0.1";
-const port = process.env.RPC_PORT || 8545;
 
 module.exports = {
   networks: {
     development: {
-      host: host,
-      port: port,
+      host: process.env.RPC_HOST || "127.0.0.1",
+      port: process.env.RPC_PORT || 8545,
       network_id: "*" // match any network
     }
-  }
+  },
+  migrations_directory: "./migrations"
```

Comment:
Isn't this related to `truffle-solidity-loader`? Probably good to keep those changes separate.

Reply:
Yup, updating